src/main/java/seedu/address/model/person/Meeting.java

```diff
@@ -0,0 +1,98 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.text.SimpleDateFormat;
+import java.time.LocalDateTime;
+
+/**
+ * Represents a Person's meetings in the Meetings field.
+ * Guarantees: immutable
+ */
+public class Meeting {
+    public static final String MESSAGE_CONSTRAINTS_LOCATION = "Location can take any values, "
+            + "and it should not be blank";
+
+    public static final String MESSAGE_CONSTRAINTS_TIME = "Start time must be before end time";
+
+    /*
+     * The first character of the location must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String VALIDATION_REGEX = "[^\\s].*";
```

Comment:
Just wondering, while I don't think we need to cater to every user's possible needs or even mistakes, if our command is 
`schedule (id) /start (some time) /end (some time) /location (some place)`
then isn't is quite understandable if the user double spaces after typing location? Shouldn't we just trim the string? and thus removing the need to regex the location string of a leading whitespace?

Reply:
Hmph good catch, I think that's fair, we can KIV for acceptable formats in the extension maybe...

Reply:
Hold on, I do think this should be in extension, because all of our other fields check for whitespaces tho, so if we allow for that in location, we should allow for every other fields too.